### PR TITLE
Update VSCode settings and suppress some Flake8 linting warnings

### PR DESCRIPTION
**Pull request by TheVinhLuong102**
Hi David & Laurens,

Some light changes for your considerations, please -- mainly to improve the experience of developers exploring this library's source code on VSCode. :)

Thank you.

Vinh

### CODE REVIEW

File: src/pybricks/parameters.py [hunk 1]
```diff
@@ -36,7 +36,7 @@
 
 
 class _PybricksEnumMeta(type(Enum)):
-    def __dir__(cls):
+    def __dir__(cls):   # noqa: N805
```
Comment (dlech) [inline src/pybricks/parameters.py:39]: If this is a class method, shouldn't we just decorate it with `@classmethod`?

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,4 +1,10 @@
 {
+    "editor.rulers": [88],
```
Comment (dlech) [inline .vscode/settings.json:2]: Since Black automatically takes care of wrapping lines, I think we should just disable E501 and not force a specific ruler size.

File: .vscode/settings.json [hunk 3]
```diff
@@ -1,4 +1,10 @@
...
+
+    "files.exclude": {
+      "**/__pycache__": true,
```
Comment (dlech) [inline .vscode/settings.json:5]: Why is this needed?

Comment (TheVinhLuong102) [reply .vscode/settings.json:5]: @dlech the __pycache__/*.pyc file shows up in the VSC browser when the developer (e.g., I) uses the library in the dev/editable mode. I think it's a good idea to hide these generated files from the developer.

File: src/pybricks/parameters.py (hunk 1 above)
Comment (TheVinhLuong102) [reply src/pybricks/parameters.py:39]: @dlech I thought so too but was not confident of its full implication to the lib. Shall I just decorate with `@classmethod` and then leave end-to-end testing to you?

File: .vscode/settings.json (hunk 2 above)
Comment (TheVinhLuong102) [reply .vscode/settings.json]: I agree, and have just removed `editor.rulers`.

File: src/pybricks/parameters.py (hunk 1 above)
Comment (dlech) [reply src/pybricks/parameters.py]: Yes, that sounds good.

File: src/pybricks/_common.py [hunk 4]
```diff
@@ -335,7 +335,7 @@ def __init__(
                 Choose ``False`` to keep the
                 current value, so your program knows where it left off last
                 time.
-        """
+        """   # noqa: E501
```
Comment (dlech) [inline src/pybricks/_common.py:338]: Instead of repeating this multiple times, let's just add `E501` to the `ignore` list in `setup.cfg` and remove all of the `noqa: E501`s.

Comment (TheVinhLuong102) [reply src/pybricks/_common.py]: done! -- yes, this is consistent with your use of Black mentioned above.

**Comment by dlech** [comment]
Merged, thanks!
